_backend/config.example.php

```diff
@@ -41,4 +41,5 @@
     'twitter_access_token'    => 'test_atoken',
     'twitter_access_secret'   => 'test_asecret',
 
+    'covid_estimate' => 'Current estimates are 13–16 business days for apparel shipping from the US, 4–7 business days for apparel shipping from the EU, 6–9 business days for mugs shipping from the US, and 2–4 business days for mugs shipping from the EU.',
```

Comment:
This includes the latest relevant estimates from https://www.printful.com/covid-19. I expect we may need to iterate on these from time-to-time as estimates change.